extension-base/src/main/java/com/azure/autorest/extension/base/plugin/JavaSettings.java

```diff
@@ -200,7 +200,13 @@ private JavaSettings(Map<String, Object> modelerSettings,
                          boolean requireXMsFlattenedToFlatten,
                          String clientFlattenAnnotationTarget,
                          String keyCredentialHeaderName,
-                         boolean skipFormatting)
+                         boolean skipFormatting,
+                         Map<String, String> llcProperties,
+                         boolean llcGenerateNonCode,
+                         boolean llcGeneratePom,
+                         boolean llcGenerateChangelog,
+                         boolean llcGenerateReadme,
+                         boolean llcGenerateSampleReadme)
```

Comment:
Why not put them to `llcProperties` as well?

Reply:
Only keep `llcGenerateNonCode` now